typings/response.d.ts

```diff
@@ -1,14 +1,34 @@
 
-export interface BranchSummary {
-   detached: boolean;
-   current: string;
-   all: string[];
-   branches: {[key: string]: {
-      current: string,
-      name: string,
-      commit: string,
-      label: string
-   }};
+export interface BranchDeletionSummary {
+	branch: string;
```

Comment:
Please can this file use the white-space rules from the `.editorconfig` - the project is set to 3 spaces for indentation rather than a tab.